src/test/java/com/commercetools/emailprocessor/model/StatisticsTest.java

```diff
@@ -25,14 +25,13 @@ public void update_differentResponseCodesAreProvided_shouldShowCorrectStatistic(
         statistics.update(Statistics.RESPONSE_CODE_SUCCESS);
         statistics.update(Statistics.RESPONSE_ERROR_TEMP);
         statistics.update(Statistics.RESPONSE_ERROR_PERMANENT);
-        statistics.update(0);
-        statistics.update(0);
+
```

Comment:
Why are these lines removed?

Reply:
Because the state "0" doesn`t exist anymore